src/Compilers/CSharp/Portable/Binder/BuckStopsHereBinder.cs

```diff
@@ -239,6 +239,11 @@ internal override BoundStatement BindLockStatementParts(BindingDiagnosticBag dia
             throw ExceptionUtilities.Unreachable;
         }
 
+        internal override FieldSymbol? GetSymbolForPossibleFieldKeyword()
+        {
+            throw ExceptionUtilities.Unreachable;
```

Comment:
Consider clarifying what types of binders should be implementing this, like the othe methods above.